src/components/Title.tsx

```diff
@@ -0,0 +1,16 @@
+type stringTitle = {
+  Titlename: string;
+};
+
```

Comment:
```suggestion
interface TitleProps = {
  title: string;
};

```

and then change the rest of your code accordingly